apps/web/hooks/escrow/useEscrow.ts

```
@@ -0,0 +1,211 @@
+        if (transformed) setStatus(transformed)
+      }
+    } catch (err) {
+      setError(
+        err instanceof Error ? err : new Error('Failed to fetch escrow status'),
+      )
+    } finally {
+      setLoading(false)
+    }
+  }
+
+  useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: fetchEscrowStatus

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)